darwin/client.py

```diff
@@ -1474,3 +1482,19 @@ def api_v2(self) -> BackendV2:
         if not team:
             raise ValueError("No team was found.")
         return BackendV2(self, team.slug)
+
+    def get_team_properties(self, team_slug: Optional[str] = None, full: bool = False) -> List[FullProperty]:
```

Comment:
Not big, but I think the bool here should be default True, I think in most instances people will want the options and certainly for our use of it, it'll be required. Also I would probably rename it so that it's more clear what it does, like include_options.